atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/Blacklist.java

```diff
@@ -17,47 +17,107 @@
 package com.palantir.atlasdb.keyvalue.cassandra;
 
 import java.net.InetSocketAddress;
+import java.util.Collection;
+import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.collect.ImmutableSet;
 import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+import com.palantir.atlasdb.cassandra.CassandraKeyValueServiceConfig;
 import com.palantir.logsafe.SafeArg;
+import com.palantir.logsafe.UnsafeArg;
 
-public class Blacklist {
+class Blacklist {
     private static final Logger log = LoggerFactory.getLogger(CassandraClientPool.class);
 
-    private Map<InetSocketAddress, Long> blacklistedHosts = Maps.newConcurrentMap();
+    private final CassandraKeyValueServiceConfig config;
+    private Map<InetSocketAddress, Long> blacklist;
```

Comment:
This is more of a style question -- isn't it better to declare it as a ConcurrentMap to make it more obvious that it is threadsafe?